chat-client/src/client/mynahUi.ts

```diff
@@ -6,30 +6,23 @@ import { MynahUI, NotificationType } from '@aws/mynah-ui'
 import { SendToPromptParams } from '../contracts/uiContracts'
 import { Messager } from './messager'
 import { TabFactory } from './tabs/tabFactory'
-import { TabStorage } from './tabs/tabStorage'
 
 export interface InboundChatApi {
     sendToPrompt(params: SendToPromptParams): void
 }
 
-export const createMynahUI = (messager: Messager, tabFactory: TabFactory, tabStorage: TabStorage): InboundChatApi => {
-    const mynahUI = new MynahUI({
+export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [MynahUI, InboundChatApi] => {
+    const mynahUi = new MynahUI({
         onReady: messager.onUiReady,
         onTabAdd: (tabId: string) => {
-            mynahUI.updateStore(tabId, {})
-            tabStorage.addTab({
-                id: tabId,
-                isSelected: true,
-            })
+            mynahUi.updateStore(tabId, {})
```

Comment:
This doesn't do anything with the tab in the store (yet, until it's filled in with some data)

Reply:
I can remove it for now. In the future, it will be used for quick actions at least.